samples/grpc/restaurant-drone-deliveries-service-java/pom.xml

```diff
@@ -22,8 +22,8 @@
         <akka-persistence-r2dbc.version>1.2.0-M3</akka-persistence-r2dbc.version>
         <akka-management.version>1.4.0</akka-management.version>
         <akka-diagnostics.version>2.0.1</akka-diagnostics.version>
-        <akka-grpc.version>2.3.3</akka-grpc.version>
-        <akka-grpc-maven-plugin.version>2.3.3</akka-grpc-maven-plugin.version>
+        <akka-grpc.version>2.3.4</akka-grpc.version>
+        <akka-grpc-maven-plugin.version>2.3.4</akka-grpc-maven-plugin.version>
```

Comment:
same with the bom here, I'd rather keep them in sync. You could just skip the pom.xml files for now.